cmd/rootMain.go

```diff
@@ -187,6 +187,7 @@ func rootMain(cmd *cobra.Command, args []string) exitCode {
 		cmdutil.ErrPrintf("Unable to start server: %v\n", err)
 		return exitCode{1}
 	}
+	registerInvocationToGA(cmd, socketpath)
```

Comment:
Having these run over the socket seems weird, and results in an extra journal entry for starting the Wash shell.

Reply:
Ah yeah, wanted to maintain the symmetry with the other commands. I guess we could also say "Server start" => "Someone is using Wash", so that this stuff could move to the `internal/server` package.